### PR TITLE
added --help to documentation

### PR DESCRIPTION
**Pull request by maltfield**
added --help to documentation. See also

 * https://github.com/LemmyNet/lemmy-stats-crawler/issues/6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ Crawls Lemmy instances using nodeinfo and API endpoints, to generate a list of i
...
 ```
 cargo run -- --start-instances baraza.africa,lemmy.ml
-```
\ No newline at end of file
+```
+
+For a complete list of arguments, use `--help`
+
+```
+cargo run -- --help
```
Comment (dessalines) [inline README.md:16]: For single lines, you can just do `cargo run -- --help`

**Comment by Nutomic** [comment]
I meant to copy to the full output of `--help` and paste it in the readme. Seems more helpful to see the options before downloading/compiling the project.

**Comment by maltfield** [comment]
oh, I didn't understand.

I intentionally didn't include the full output because it'll be eventually stale. I figured it's better to tell the user how to get the info from a single authoritative place than to try to maintain it in two distinct locations.

Is this a blocker for the PR merge?

**Review by Nutomic — APPROVED** [review]

**Comment by Nutomic** [comment]
Alright lets do it this way.

**Pull request merged**